.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,95 @@
+name: Bug Report
+description: Something isn't working as expected?
+labels:
+  - Bug
+  - triage
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### ⚠️ This bug report is public
+        Please create a reproducible example using this [CodeSandbox Template](https://codesandbox.io/s/carbon-quickstart-xi5jc). 
+        Please do not include any confidential or commercially sensitive information.
+        
+        ---
+  - type: textarea
+    id: current-behaviour
+    attributes:
+      label: Current behaviour
+      description: |
+        Please give a clear and concise description of the current behaviour.
+        If applicable, add screenshots or screen recording of your CodeSandbox to help explain the problem. You can paste these directly into GitHub.
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behaviour
+    attributes:
+      label: Expected behaviour
+      description: |
+        Please give a clear and concise description of the current behaviour.
+    validations:
+      required: true
+  - type: input
+    id: codesandbox-url
+    attributes:
+      label: CodeSandbox URL
+      description: Please fork [this template](https://codesandbox.io/s/carbon-quickstart-xi5jc), and ensure you save your changes.
+      placeholder: https://codesandbox.io/s/carbon-quickstart-xi5jc
+    validations:
+      required: true
```

Comment:
Do we want to make having a codesandbox example required? Sometimes users find bugs in Storybook so the codesandbox would be redundant then.

Reply:
I'll make this Codesandbox or Storybook :)

Reply:
Excellent. That'll be splendid 😃